recipes/variant-effect-predictor/build.sh

```diff
@@ -29,3 +29,4 @@ rm -rf t/
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/CADD.pm
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/dbNSFP.pm
 wget --no-check-certificate https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm
```

Comment:
What is the `right commit`? The problem is that you don't know what you get with the current approach. You will get the script from the time of build. If you rebuild the package you can get an entire new script in worst case. With fixing a revision you are guaranteed to at least get every time the same script.